release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1305,6 +1305,18 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 [id="ocp-4-14-dev-console-bug-fixes"]
 ==== Developer Console
 
+* Previously, if you tried to edit a Helm chart repository in the *Developer* console by navigating to *Helm*, clicking the *Repositories* tab, then selecting *Edit HelmChartRepository* through the kebab menu for your Helm chart repository, an *Error* page was displayed that showed a *404: Page Not Found* error. This was caused by a component path that was not up to date. This issue is now fixed. (link:https://issues.redhat.com/browse/OCPBUGS-14660[*OCPBUGS-14660*])
+
+* Previously, it was difficult to distinguish between the types of samples listed in the *Samples* page. With this fix, you can easily identify the sample type from the badges displayed on the *Samples* page. (link:https://issues.redhat.com/browse/OCPBUGS-7446[*OCPBUGS-7446*])
+
+* Previously, on the Pipeline *Metrics* page only four legends are visible for TaskRun duration charts. With this update, you can see all the legends present for the TaskRun duration charts. (link:https://issues.redhat.com/browse/OCPBUGS-19878[*OCPBUGS-19878*])
+
+* Previously, an issue occurred when creating an application using the `Import JAR` form in a disconnected cluster with the Cluster Samples Operator not installed. With this update, the `Import JAR` form from the *Add* page and the *Topology* page. (link:https://issues.redhat.com/browse/OCPBUGS-15011[*OCPBUGS-15011*])
```

Comment:
Is the 2nd sentence missing something? Or maybe I'm reading it wrong.

Reply:
fixed.